docs/templates.rst

```diff
@@ -640,9 +640,12 @@ iterate over containers like `dict`::
     {% endfor %}
     </dl>
 
-Note, however, that **Python dicts are not ordered**; so you might want to
-either pass a sorted ``list`` of ``tuple`` s -- or a
-``collections.OrderedDict`` -- to the template, or use the `dictsort` filter.
+Note, however, that **Python dicts preserve insertion order**, which may not
```

Comment:
May be plausible to remove the _however_ and already say at the start, that this is only guaranteed since 3.7